modules/reference/pages/properties/cluster-properties.adoc

```diff
@@ -5542,7 +5542,7 @@ Timeout to wait for leadership in metadata cache.
 
 === write_caching_default
 
-The default write caching mode to apply to user topics. Write caching acknowledges a message as soon as it is received and acknowledged on a majority of brokers, without waiting for it to be written to disk. With `acks=all`, this provides lower latency while still ensuring that a majority of brokers acknowledge the write. Fsyncs follow <<`raft_replica_max_pending_flush_bytes`>> and <<`raft_replica_max_flush_delay_ms`>>, whichever is reached first.
+The default write caching mode to apply to user topics. Write caching acknowledges a message as soon as it is received and acknowledged on a majority of brokers, without waiting for it to be written to disk. With `acks=all`, this provides lower latency while still ensuring that a majority of brokers acknowledge the write. Fsyncs follow <<raft_replica_max_pending_flush_bytes,`raft_replica_max_pending_flush_bytes`>> and <<raft_replica_max_flush_delay_ms,`raft_replica_max_flush_delay_ms`>>, whichever is reached first.
```

Comment:
This one I'm not clear about because I don't know adoc syntax, will this make the << >> go away?

Reply:
n/m I see there's a preview in the PR and that looks good, thanks @asimms41 !

Reply:
You can see it in the preview here: [topic preview](https://deploy-preview-580--redpanda-docs-preview.netlify.app/current/reference/properties/cluster-properties/#write_caching_default)